install/updateVersion.sh

```diff
@@ -54,7 +54,11 @@ function gen_file() {
     fi
 }
 
-for target in istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-namespace-auth.yaml istio-multicluster.yaml istio-auth-multicluster.yaml istio-remote.yaml;do
+TARGETS="istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-namespace-auth.yaml"
```

Comment:
Please don't add more deprecated files. updateVersion is only for the legacy files and for 0.8 transition, will go away.
Helm and 'set values' should be used consistently. 

Not sure even if we'll ship updateVersion.sh in 0.8.